client/my-sites/email/email-providers-stacked-comparison/provider-cards/provider-card-props.ts

```diff
@@ -1,26 +1,23 @@
 import { ReactElement } from 'react';
```

Comment:
This should only import the type.
```suggestion
import type { ReactElement } from 'react';
```